mobile/android/qt6/build.gradle

```diff
@@ -87,8 +87,6 @@ android {
         minSdkVersion qtMinSdkVersion
         targetSdkVersion qtTargetSdkVersion
         ndk.abiFilters = qtTargetAbiList.split(",")
-        // Generate unique versionCode based on minutes since epoch
-        versionCode = (System.currentTimeMillis() / 60000).toInteger()
```

Comment:
you cant remove this, this is important

Reply:
It's not removed, just moved here to follow the same logic (build number) for both platforms:

```
ANDROID_VERSION_CODE = $$(BUILD_VERSION)
```

https://github.com/status-im/status-desktop/pull/19311/files#diff-3d5d87b2e5d5d132c4be7cf47d6d05ad594fcd6fba24405675aa0401356e1dbdR28

Reply:
The machinery is explained here:
https://doc.qt.io/qt-6/android-manifest-file-configuration.html#qt-default-configuration